### DISCUSSION TITLE
Protected tweets

### DISCUSSION THREAD

**Discussion by Linktydraa** [TumblThreeApp/TumblThree#423] [General]
Hi all,

On some of the twitter pages I try to download, I get the error message "Protected of offline blog detected". I went to the blog to make sure it was still active and it is. I'm also following the user, so I'm not sure if the tweets are protected or not.

My question is why doesn't Tumblethree bypass the protected twet status?

Why isn't there a login option for Twitter on tumblethree like there is for Tumblr in order to bypass protected tweets?

**Comment by thomas694** [comment]
Hi,

as you've already found out, the blogs are accessible when you access them through the browser and are logged in. But TumblThree can only download publicly accessible ones by now.

There's no login option, because no one has implemented it yet. Volunteers are welcome.
